reverbePrintemps/spiced
# Attach functionality to calculator

# Context

This PR aims to tackle [the simple calculator challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-basics/challenges-js-basics.md#simple-calculator).

# Changes

- [x] Redefine operand1 as let
- [x] Perform expected mathematical operation within each event listener

# How to run

1. Clone repo
2. Checkout this branch
3. Run page in browser

# Test

- Open browser inspector
- Press each button on the calculator and notice expected result being logged in the console

# TODO


# GIF

![](https://media.giphy.com/media/UvOaWabxo1enu/giphy.gif)
